service_mesh/service_mesh_user_guide/ossm-observability.adoc

```diff
@@ -0,0 +1,27 @@
+[id="ossm-observability"]
+= Data visualization and observability 
+include::modules/ossm-document-attributes.adoc[]
+:context: observability
+
+toc::[]
+
+You can view your application's topology, health and metrics with using the Kiali operator you installed as part of your {ProductName} installation. 
+
+If your service is having issues, Kiali offers ways to visualize the data flowing through your service. Kiali offers insights about the mesh components at different levels, including abstract applications, services, and workloads. It also provides an interactive graph view of your namespace in real time. 
+
+.Prerequisites
+
+* You can observe data flowing through a working service if you install the xref:../../service_mesh/service_mesh_day_two/ossm-example-bookinfo.html#ossm-tutorial-bookinfo-overview_ossm-bookinfo-tutorial[Bookinfo sample application]. 
```

Comment:
Are you trying to show the traffic animation here?  Users might need to select that checkbox in the menu to see the movement.  I don't think it's selected by default.

Reply:
No, I'm not explaining the animation here. I'm adding the bookinfo example app as a prereq because it's useful for explaining the concepts later.

Reply:
The phrase "data flowing" was what made me ask.  Maybe re-phase as just "observe the data flow through a service" to remove that impression of movement?

Reply:
done